library/src/test/java/com/chuckerteam/chucker/internal/support/ShareUtilsTest.kt

```diff
@@ -35,6 +37,10 @@ class ShareUtilsTest {
         every { getString(R.string.chucker_body_empty) } returns "(body is empty)"
     }
 
+    private val requestMethods = arrayOf("GET", "POST", "PUT", "DELETE")
+
+    private fun getRandomHttpMethod(): String = requestMethods[Random.nextInt(0, 4)]
```

Comment:
I know you're just moving the code so feel free to drop, but:

There is a `.random()` extension function on `Array` that we can use here:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/random.html

Reply:
I will update. Thanks.